cardano-testnet/test/cardano-testnet-test/files/golden/queries/govStateOut.json

```diff
@@ -358,16 +358,16 @@
                 38887044,
                 32947,
                 10,
-                9223372036854775807,
-                9223372036854775807,
-                9223372036854775807,
-                9223372036854775807,
-                9223372036854775807,
-                9223372036854775807,
-                9223372036854775807,
-                9223372036854775807,
-                9223372036854775807,
-                9223372036854775807
+                1292075,
```

Comment:
I don't know sure what's causing these changes. What I did in this PR should have no runtime effect, so I'm wondering.

@carbolymer> would you have a clue?

Reply:
I haven't read your code thoroughly but you're using `data-default-class`, maybe it pulled a sane plutus v2 cost model default from ledger instead of 2^63-1?

Reply:
OK found from where the data comes. It's those costs that are showing up now: https://github.com/IntersectMBO/cardano-api/blob/4b3e3305e8bcb5b7eef33cdd1baf1b93d330a7b2/cardano-api/internal/Cardano/Api/Genesis.hs#L726

Still don't know what causes this though

Reply:
OK found it: https://github.com/IntersectMBO/cardano-node/pull/5976/files#r1749955020

The change is correct: before, we were not honoring the user-supplied era in `cardanoTestnetDefault`.